database/schema.sql

```diff
@@ -1,210 +1,424 @@
-	for_district varchar(17) NULL,
-	paymeth_code varchar(5) NULL,
-	paymeth_type smallint NULL,
-	prev_paymeth varchar(5) NULL,
-	prev_paymeth_type smallint NULL,
-	employee_ind smallint NULL,
-	employee_no varchar(12) NULL,
-	swipe_issue_date timestamp NULL,
-	swipe_card_request smallint NULL,
-	hb_sar smallint NULL,
-	fast_track smallint NULL,
-	equal_access_code varchar(3) NULL,
-	sched_code varchar(6) NULL,
-	person_status varchar(1) NULL,
-	transmission_date timestamp NULL,
-	lead_liab_title varchar(8) NULL,
-	lead_liab_forename varchar(32) NULL,
-	lead_liab_middlename varchar(32) NULL,
-	lead_liab_surname varchar(32) NULL,
-	lead_consent_ind smallint NULL,
-	for_addr_last_updated int NULL,
-	hb_archive_ind smallint NULL,
-	party_ref int NULL,
-	for_addr_verified smallint NULL,
-	last_updated_int int NULL,
-	prohibit_code varchar(2) NULL,
-	prohibit_end timestamp NULL,
-	prohibit_set timestamp NULL,
-	prohibit_user varchar(8) NULL,
-	prohibit_online_spd smallint NULL,
-	for_addr_abroad smallint NULL,
-	documerge_excl_ind smallint NULL
-);
-
-CREATE TABLE ctproperty (
-	division_id smallint NULL,
-	property_ref varchar(18) NULL,
-	fund_no smallint NULL,
-	parish_code varchar(8) NULL,
-	street_code varchar(8) NULL,
-	house_id varchar(8) NULL,
-	addr1 varchar(35) NULL,
-	addr2 varchar(35) NULL,
-	addr3 varchar(32) NULL,
-	addr4 varchar(32) NULL,
-	postcode varchar(8) NULL,
-	grid_ref varchar(20) NULL,
-	old_property_ref varchar(20) NULL,
-	property_type varchar(40) NULL,
-	owner_name varchar(32) NULL,
-	own_addr1 varchar(32) NULL,
-	own_addr2 varchar(32) NULL,
-	own_addr3 varchar(32) NULL,
-	own_addr4 varchar(32) NULL,
-	own_postcode varchar(8) NULL,
-	owner_away_ind smallint NULL,
-	owner_liable smallint NULL,
-	crown_ind smallint NULL,
-	council_ha_ind varchar(1) NULL,
-	institution smallint NULL,
-	planning_ind smallint NULL,
-	planning_desc varchar(30) NULL,
-	planning_granted timestamp NULL,
-	survey_code varchar(6) NULL,
-	workarea varchar(6) NULL,
-	walking_order varchar(6) NULL,
-	deleted_ind smallint NULL,
-	deleted_eff timestamp NULL,
-	band_010493 varchar(1) NULL,
-	cc_payable numeric(14, 2) NULL,
-	transit_93 numeric(14, 2) NULL,
-	transit_94 numeric(14, 2) NULL,
-	transit_95 numeric(14, 2) NULL,
-	transit_96 numeric(14, 2) NULL,
-	transit_97 numeric(14, 2) NULL,
-	inhibit_archive smallint NULL,
-	owner_code varchar(4) NULL,
-	uprn varchar(12) NULL,
-	wardcode varchar(6) NULL,
-	owner_title varchar(8) NULL,
-	owner_forename varchar(32) NULL,
-	owner_middlename varchar(32) NULL,
-	owner_surname varchar(32) NULL,
-	inhibit_llpg smallint NULL,
-	llpg_xref_key varchar(14) NULL,
-	owner_addr_verified smallint NULL,
-	last_updated_int int NULL,
-	rented_prop smallint NULL,
-	own_telno varchar(20) NULL,
-	own_email_addr varchar(128) NULL,
-	no_beds smallint NULL,
-	annexe_ind smallint NULL,
-	main_prop_ref varchar(18) NULL,
-	own_is_charity smallint NULL,
-	own_addr_abroad smallint NULL
-);
+CREATE TABLE hbhousehold (
```

Comment:
I'm not sure why the git diff think I've changed the whole file. But the changes I actually made were the addition of the table on line 104.